core/server/web/api/index.js

```diff
@@ -0,0 +1,21 @@
+const debug = require('ghost-ignition').debug('web:api:default:app');
+const express = require('express');
+const urlUtils = require('../../services/url/utils');
+const errorHandler = require('../shared/middlewares/error-handler');
+
+module.exports = function setupApiApp() {
+    debug('Parent API setup start');
+    const apiApp = express();
+
+    // Mount different API versions
+    apiApp.use(urlUtils.getVersionPath({version: 'v0.1'}), require('./v0.1/app')());
+    apiApp.use(urlUtils.getVersionPath({version: 'v2', type: 'content'}), require('./v2/content/app')());
+    apiApp.use(urlUtils.getVersionPath({version: 'v2', type: 'admin'}), require('./v2/admin/app')());
```

Comment:
The PR looks great 👍 
Was wondering why we mount urls without trailing slash now? 🤔

Reply:
Was resolved in an extra commit 👍